src/queries/dash/pageviews.sql

```diff
@@ -0,0 +1,30 @@
+--- description: Get Helix RUM data for a given domain or owner/repo combination
+--- Authorization: none
+--- Access-Control-Allow-Origin: *
+--- limit: 10
+--- interval: 30
+--- offset: 0
+--- startdate: 2022-02-01
+--- enddate: 2022-05-28
+--- timezone: UTC
+--- url: -
+--- device: all
+--- domainkey: secret
+WITH pageviews_by_id AS (
+  SELECT
+    hostname,
+    id,
+    MAX(weight) AS pageviews
+  FROM
+    `helix-225321.helix_rum.EVENTS_V4`(
+      net.host(@url), @offset, @interval, '-', '-', 'UTC', 'all', @domainkey
```

Comment:
- `@url` should have the same semantics as the rest of the `@url` parameters, so URL prefix without `https://`, optional path and `$` or `?` delimiters.
- if you list `startdate` and `enddate` in the supported parameters, you should also pass them through here
- same for `timezone`
- same for `device`

Reply:
Also, the same data should come from `PAGEVIEWS_V4` – why does this query exist when `rum-pageviews` is there already?